src/async_impl/client.rs

```diff
@@ -3040,16 +3020,6 @@ impl Future for PendingRequest {
                 },
             };
 
-            #[cfg(feature = "cookies")]
-            {
-                if let Some(ref cookie_store) = self.client.cookie_store {
-                    let mut cookies =
-                        cookie::extract_response_cookie_headers(res.headers()).peekable();
-                    if cookies.peek().is_some() {
-                        cookie_store.set_cookies(&mut cookies, &self.url);
-                    }
-                }
-            }
```

Comment:
`ClientRef::fmt_fields` is still display `cookie_store`,  which I only found that needs these codes.